content/en/Platform Deep Dive/Scans/_index.md

```diff
@@ -32,21 +32,28 @@ We use the following IP Address to make requests to your target: **52.19.40.38**
 
 ![Add Target](/deepdive/scans/2_AddTarget.png "Add Target")<br>
 
+### Authentication
+
```

Comment:
Having headers breaks the enumeration (`3. ` next line), maybe we should take the enumeration away?